Hurd: comment PF_LINK/AF_LINK defines

Comment out the PF_LINK and AF_LINK defines, since they are usually associated
with struct sockaddr_dl, which is not available on Hurd.

2012-06-22  Pino Toscano  <toscano.pino@tiscali.it>

	* sysdeps/mach/hurd/bits/socket.h (PF_LINK): Comment out.
	(AF_LINK): Likewise.
--- a/sysdeps/mach/hurd/bits/socket.h
+++ b/sysdeps/mach/hurd/bits/socket.h
@@ -97,7 +97,7 @@ enum __socket_type
 #define	PF_HYLINK	15	/* NSC Hyperchannel protocol.  */
 #define	PF_APPLETALK	16	/* Don't use this.  */
 #define	PF_ROUTE	17	/* Internal Routing Protocol.  */
-#define	PF_LINK		18	/* Link layer interface.  */
+/* #define	PF_LINK		18	Link layer interface.  */
 #define	PF_XTP		19	/* eXpress Transfer Protocol (no AF).  */
 #define	PF_COIP		20	/* Connection-oriented IP, aka ST II.  */
 #define	PF_CNT		21	/* Computer Network Technology.  */
@@ -130,7 +130,7 @@ enum __socket_type
 #define	AF_HYLINK	PF_HYLINK
 #define	AF_APPLETALK	PF_APPLETALK
 #define	AF_ROUTE	PF_ROUTE
-#define	AF_LINK		PF_LINK
+/* #define	AF_LINK		PF_LINK */
 #ifdef __USE_MISC
 # define	pseudo_AF_XTP	PF_XTP
 #endif
